nipy/labs/viz_tools/test/test_activation_maps.py

```diff
@@ -15,6 +16,11 @@
 except ImportError:
     raise SkipTest('Could not import matplotlib')
 
+try:
+    from mock import patch
+except ImportError:
```

Comment:
ah, right -- this would be two new uncovered lines if mock is available.  Just add to `except` line above an inline comment `# pragma: no cover` which would then instruct coverage to exclude next block from its consideration.  Should bring coverage back (and show enabled codecov in action)